src/Core/Managed/Shared/Extensibility/Metrics/Metric.cs

```diff
@@ -1,13 +1,12 @@
-﻿namespace Microsoft.ApplicationInsights.Extensibility
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Metrics
```

Comment:
I am worried that this namespace change will break customers using the beta SDK with the old namespace (I know I just used this functionality in AIMC and it will break again when updating to the next SDK release).
How important is it to rename the namespace?
And are you willing to change all of the internal applications with references to the old namespace and upgrade them when the SDK releases?